pkg/gorilla/gorilla.go

```diff
@@ -189,12 +200,12 @@ func (ws *WebSocket) initialize() {
 				var res rpc.RPCResponse
 				err := ws.read(&res)
 				if err != nil {
-					// TODO need to find a proper way to log this error
+					ws.logger.Logger.Err(err)
```

Comment:
Errors should always be returned instead of logged and missed

Reply:
I'm not where to return it is an anonymous func also it is concurrent I can create a channel to return the errors but I'm not sure that will logical.

Reply:
Ah right I see... Ok lets log for now and please create an issue for this then. I will keep this unresolved until the issue is created by either of us, but it's ok to leave as is. Thanks ❤️

Reply:
https://github.com/surrealdb/surrealdb.go/issues/67